src/syscheckd/src/fim_diff_changes.c

```diff
@@ -462,7 +462,7 @@ char *fim_file_diff(const char *filename, const directory_t *configuration) {
 diff_data *initialize_file_diff_data(const char *filename, const directory_t *configuration){
     diff_data *diff;
     char buffer[PATH_MAX];
-    char abs_diff_dir_path[PATH_MAX + 1];
+    char abs_diff_dir_path[PATH_MAX - 46];
```

Comment:
I think it is more beautiful to remove the `+1` or increase the length of buffer in the snprintf

Reply:
Done